### PR TITLE
The great bar sign port of 2017

### PR DESCRIPTION
**Pull request by jknpj**
shitfuck

Ports around 32 new bar signs from paradise, tweaks THE MEN to remove a lot of white pixels, also moves europa signs to their own file.

:cl:
 * rscadd: Ports 32 new bar signs from Paradise.

### CODE REVIEW

**Comment by N3X15** [comment]
The cyber-sylph's one looks awful.

**Comment by Exxion** [comment]
Its framerate is wonky as hell

**Comment by jknpj** [comment]
I wanted it to be a little slower but i guess i went too far

**Comment by N3X15** [comment]
It's not just the framerate, it's the hard edges on the alpha.  If you made the "scanlines" just have like 50% alpha, and the rest of the image 75%, it would look better.

**Comment by Shadowmech88** [comment]
The adminbus and coderbus signs are a little too close to the OOC line for my taste.

**Comment by kol1th** [comment]
MY IMULSIONS

File: code/game/objects/structures/barsigns/paradise.dm [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+	name = "The Coderbus"
+	icon = "thecoderbus"
+	desc = "A very controversial bar known for its wide variety of constantly-changing drinks."
+
+/datum/barsign/paradise/theadminbus
+	name = "The Adminbus"
+	icon = "theadminbus"
+	desc = "An establishment visited mainly by space-judges. It isn't bombed nearly as much as court hearings."
+
+/datum/barsign/paradise/oldcockinn
```
Comment (kol1th) [inline code/game/objects/structures/barsigns/paradise.dm:41]: ruh roh

**Comment by jknpj** [comment]
But @Shadowmech88 @kol1th 

wish you were here

**Comment by Fruktlimpa** [comment]
Combo Cafe looks sick tbh with you fams

**Comment by Riekland** [comment]
Why are there two beerskys, and why is it 4 the emprah instead of 4 teh emprah?

**Comment by ihadtoregisterforthis** [comment]
The Cyber Sylph sign looks one hundred percent out of place

**Comment by jknpj** [comment]
@Riekland i think i remove the second beers- oh fuck i didn't, i don't know about the emprah, i didn't make it.

**Comment by CptWad** [comment]
Yeah not a fan of the adminbus and coderbus

**Comment by yclatious** [comment]
No to sylph coder and bus, rest are fine.

File: code/game/objects/structures/barsigns/paradise.dm [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+/datum/barsign/paradise/spaceasshole
+	name = "Space Asshole"
+	icon = "spaceasshole"
+	desc = "Open since 2125, Not much has changed since then; the engineers still release the singulo and the damn miners still are more likely to cave your face in that deliver ores."
+
+/datum/barsign/paradise/syndibarsign
+	name = "The Syndi Cat"
+	icon = "syndibarsign"
+
+/datum/barsign/paradise/empbarsign
```
Comment (Kurfursten) [inline code/game/objects/structures/barsigns/paradise.dm:147]: >didn't actually change the emp_act on the bar sign object to make it change to this

C'mon

**Comment by Probe1** [comment]
Looks good to me. @jknpj is it good to go?

**Comment by jknpj** [comment]
Hold on i have the emp_act done, give me a second

**Pull request merged**